fbs/response.fbs

```diff
@@ -0,0 +1,21 @@
+include "router.fbs";
+include "transport.fbs";
+include "webRtcServer.fbs";
+include "worker.fbs";
+
+namespace FBS.Response;
+
+union Body {
+  FBS.Router.RouterDump,
+  FBS.Transport.TransportDump,
+  FBS.Transport.ConsumeResponse,
```

Comment:
I don't see a clear naming pattern here. Why ConsumeResponse and then WorkerDump? IMHO it should be:

FBS.Worker.Dump.Response

Otherwise there is no naming pattern. 

Also, there are missing requests/responses yet such as FBS.Consumer.Dump.Response, etc right?

Reply:
For consistency I can make them `{method}Response`:

RouterDumpResponse, TransportDumpResponse, ConsumerResponse, etc.

> FBS.Worker.Dump.Response

Again, this would require creating a schema file for each request, and it's IMHO not more readable than `FBS.Transport.RouterDumpResponse`

Reply:
fixed.